app/models/tax_svc.rb

```diff
@@ -6,23 +6,33 @@
 require 'logging'
 
 class TaxSvc
+  READ_TIMEOUT_ERROR = RestClient::Exceptions::ReadTimeout
+  OPEN_TIMEOUT_ERROR = RestClient::Exceptions::OpenTimeout
+
   def get_tax(request_hash)
     log(__method__, request_hash)
     RestClient.log = logger.logger
+    order_number = request_hash[:DocCode]
     res = response('get', request_hash)
     logger.info 'RestClient call'
     logger.debug res
     response = JSON.parse(res.body)
 
     if response['ResultCode'] != 'Success'
-      logger.info_and_debug("Avatax Error: Order ##{request_hash[:DocCode]}", response)
+      logger.info_and_debug("Avatax Error: Order ##{order_number}", response)
       raise 'error in Tax'
     else
       response
     end
-  rescue => e
-    logger.info 'Rest Client Error'
-    logger.debug e, 'error in Tax'
+  rescue StandardError => e
+    # UDL-946 - Notify a failure to calculate taxes for an order
+    if [READ_TIMEOUT_ERROR, OPEN_TIMEOUT_ERROR].any? { |klass| e.instance_of?(klass) }
+      message = "[#{Rails.env}] Total Tax 0.0 calculated for Order: #{order_number}. Error: #{e}."
+      Slack_client.chat_postMessage(channel: 'mejuri-web-stripe-v3-alerts', text: message)
+    end
+    msg = "Rest Client Error for Order ##{order_number}. Error: #{e.class}"
+    logger.info msg
+    logger.debug e, msg
```

Comment:
Another nitpick: can this be just one error log instead of info/debug?

Reply:
Hi Pablo! I agree that it should be one level only, the problem is the method was constructed to raise an exception when Avalara returns 0 taxes as well [here](https://github.com/mejuri-inc/spree_avatax_certified/blob/e556222bd9fb6196c29d1458f1f8e9c6647a58aa/app/models/tax_svc.rb#L23). 

Initially the message did not have the error class, maybe now that it does it can help us differentiate actual errors from the case where Avalara returns 0 as a tax (which is a valid business scenario for some addresses)

Anyway, I will add is as `error` thanks!